Formula/talisman.rb

```diff
@@ -0,0 +1,20 @@
+class Talisman < Formula
+  desc "Tool to detect and prevent secrets from getting checked in"
+  homepage "https://thoughtworks.github.io/talisman/"
+  url "https://github.com/thoughtworks/talisman",
+      :using    => :git,
+      :tag      => "v1.0.0",
+      :revision => "dd7b5898f7c4d9013d23d4dbd34685611524ac20"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+
+  def install
+    system "./build"
```

Comment:
Does this pull in dependencies?

Reply:
I've removed the explicit dependencies on `gox`, since './build' will pull in the dependencies as well

Reply:
build dependencies on gox is need, build script is also bringing in the gox but since its on path, `./build` not able to find it.